phpdotnet/phd/Package/PHP/Web.php

```diff
@@ -211,6 +223,40 @@ protected function writeJsonIndex() {
         v("Index written", VERBOSE_FORMAT_RENDERING);
     }
 
+    protected static function generateDeprecatedInfo($filename) {
+        static $info;
```

Comment:
It seems like this function can (should?) be merged with generateVersionInfo() and both be done at the same time, rather the iterating over the file twice